tensorflow/contrib/ios_examples/simple/tf_ios_makefile_example.xcodeproj/project.pbxproj

```diff
@@ -302,14 +306,18 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = "com.google.TF-Test";
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SEPARATE_STRIP = NO;
 			};
 			name = Debug;
 		};
 		591157B41CF4011D00C31E3A /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				CLANG_DEBUG_INFORMATION_LEVEL = default;
```

Comment:
> For debug_information_level, is the default for Release essentially 'none'?

Yes, that's correct.

> exceptions and rtti are not needed by tensorflow and at least rtti adds code size .
> (although maybe rtti is needed if we are not using proto lite runtime here).

I've turned off exceptions (though not rtti) for the protobuf and tensorflow library builds. I found I hit problems when trying it with the actual app though, but I'm guessing since it's only a small amount of code it wouldn't make a massive difference. Disabling exceptions on TF and protobuf did help a lot though.

Thanks for the comment though, I'd forgotten that RTTI was separate from exceptions, so I can try disabling that too!